node/src/chain_spec/turing.rs

```diff
@@ -105,6 +105,10 @@ fn testnet_genesis(
 	general_councils: Vec<AccountId>,
 	technical_memberships: Vec<AccountId>,
 ) -> turing_runtime::GenesisConfig {
+	let candidate_stake = std::cmp::max(
```

Comment:
why do max for the two? shouldn't MinCandidateStk always be larger? Or is it because of some weird edgecase in the runtime where MinCandidateStk could not yet be set and therefore be 0?

Reply:
Generally, MinCandidateStk is lower that MinCollatorStk. The CandidateStk is the amount that the collator has to put up themselves, while the CollatorStk is the minimum required to be selected to author blocks. A Collator's Stk includes their CandidateStk + any delegations so generally it should be a larger number. The relationship is thrown off a bit for us because we've allowed some collators to join the candidate pool at a reduced number in the closed sale and we want to make sure that they can still produce blocks once this goes live (so MinCollatorStk has to stay at the lower number.